src/style.css

```diff
@@ -1 +1,64 @@
 /* Styles go here */
+body {
+  margin: 0%;
+}
+
+a:active {
+  color: orange;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: white;
+}
+
+.logo_img {
+  display: block;
+  padding: 10px;
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0%;
+  padding: 0%;
+  list-style: none;
+}
+
+.nav {
+  display: inline-block;
+}
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  text-decoration: none;
+  margin: 0 20px;
+  color: black;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  letter-spacing: 0 em;
+  text-align: left;
+  height: 60px;
+  line-height: 60px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background: #00acdc;
```

Comment:
make variable for colors which uses more than once

Reply:
<img width="647" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/c59305c6-665f-42c7-b44d-8b0318ceb953">
use the blue var here